lib/hex_web/owners.ex

```diff
@@ -0,0 +1,62 @@
+defmodule HexWeb.Owners do
```

Comment:
I am afraid `Owners` is too easy to misspell. I prefer `OwnerCrud` and we can put it under `web/crud/owner_crud.ex`.